src/function.rs

```diff
@@ -25,6 +26,10 @@ lazy_static! {
     ("os_family", Nullary(os_family)),
     ("parent_directory", Unary(parent_directory)),
     ("without_extension", Unary(without_extension)),
+    ("uppercase", Unary(uppercase)),
```

Comment:
Ahh, sorry I forgot, this list should be sorted. Also, the functions themselves should also be declared in alphabetic order in this file.